src/Chapter1.hs

```diff
@@ -652,9 +668,13 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
-
+firstDigit :: Int -> Int
+firstDigit n
+    | npositive < 10 = npositive
+    | otherwise = 
+        let x = div npositive 10
+        in (firstDigit x)
+  where npositive = if n < 0 then n * (-1) else n
```

Comment:
It is better to use `abs` function in here as well 🙂 
Also, you don't need parenthesis after `in` keyword.